src/Symfony/Component/Uid/Uuid.php

```diff
@@ -74,7 +74,7 @@ public static function fromString(string $uuid): parent
             case NilUuid::TYPE: return new NilUuid();
         }
 
-        return new self($uuid);
+        return new static($uuid);
```

Comment:
This place can only be reached if the actual class is Uuid so it is safe here, there is nothing to change.

Reply:
reverted